src/applications/simple-forms/20-10207/tests/unit/components/FileField.unit.spec.jsx

```diff
@@ -1502,4 +1502,95 @@ describe('Schemaform <FileField>', () => {
       expect(individualFileTryAgainButton).to.not.exist;
     });
   });
+
+  describe('functional file field tests', () => {
```

Comment:
Oh are we testing all the functions together? I would have assumed each would get their own test since presumably their functionality differs. I haven't read deeply into what's here but if you're good with it, so am I.

Reply:
update: removed tests and local FileField altogether, to favor forms-system FileField